### PR TITLE
Add conda as installing method

### LINKED ISSUE #3163: Installation using conda
In the section [Quick start](https://github.com/great-expectations/great_expectations/blob/develop/README.md#quick-start) on the `README.md` one could add the possibility to install via conda (see [link](https://anaconda.org/conda-forge/great-expectations)) via `conda install -c conda-forge great-expectations`

### PR DESCRIPTION
**Pull request by rpanai**
see [this](https://github.com/great-expectations/great_expectations/issues/3163)

Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

Thank you for submitting!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,10 +128,18 @@ Quick start
...
+You can install it using pip
 ```
     pip install great_expectations
-    great_expectations init
+```
+or conda
+```
+conda install -c conda-forge great-expectations
+```
+and then run
```
Comment (cdkini) [inline README.md:139]: Screen Shot 2021-08-09 at 10 33 18 AM
Could you please left align all of these commands?

**Review by cdkini — CHANGES_REQUESTED** [review]
Hey @rpanai thanks so much for opening this PR! Definitely a valuable option for our `conda` users.

I requested one minor change; if you could fix that, I'll happily merge this in :)

File: README.md (hunk 1 above)
Comment (rpanai) [reply README.md:139]: done

**Review by cdkini — APPROVED** [review]
Thanks for such a quick response!

Approving 🎉

**Pull request merged**